### DISCUSSION TITLE
query regarding contributions

### DISCUSSION THREAD

**Discussion by oooo-ankit** [code-chaser/project-leetcode#51] [Q&A] [answered]
Can i add any leetcode solution i have attempted so far or are you accepting solutions for only a few selected questions?
I am new to open source so apologies if I asked my doubt in the wrong category.  I want to take part in Hacktoberfest

**Comment by code-chaser** [comment, marked as answer]
You can solve any LeetCode question (3 at least in one PR)
- Please see #18 

Welcome to the open-source community! 🎉

**Reply by oooo-ankit** [reply]
Is the make and makefile compulsary? I understand that it is used to automate the files locally but cannot I run my code on leetcode and then contribute the solutions to the repository once all test cases are passed?
